### PR TITLE
login1 and RPC working on Electron for Linux + OSX

### PR DESCRIPTION
**Pull request by cjb**

### CODE REVIEW

File: react-native/react/engine/index.js [hunk 1]
```diff
@@ -14,47 +15,27 @@ const {
...
-      return new Error(JSON.stringify(err))
-    }
-  }
-
-  _raw_write_bufs (len, buf) {
-    const buffer = Buffer.concat([new Buffer(len), new Buffer(buf)])
-    const data = buffer.toString('base64')
-    this.writeCallback(data)
-  }
+let platform = 'desktop'
```
Comment (chrisnojima) [inline react-native/react/engine/index.js]: i think this was in some helper, let's discuss

**Comment by chrisnojima** [comment]
:+1: % comments

File: react-native/react/engine/index.js (hunk 1 above)
Comment (cjb) [inline react-native/react/engine/index.js]: @chrisnojima Oops, that was left over code -- we don't need a platform var anymore, just imports .mobile or .desktop now.  Removed it.

**Comment by chrisnojima** [comment]
:+1: :100:

**Pull request merged**